_inc/client/plans/index.jsx

```diff
@@ -79,6 +79,7 @@ export const Plans = React.createClass( {
 						activeFeatures={ activeFeatures }
 						siteRawUrl={ this.props.siteRawUrl }
 						siteAdminUrl={ this.props.siteAdminUrl }
+						rewindStatus={ this.props.rewindStatus }
```

Comment:
You could pass here only the true/false of the `active` property to avoid passing an entire object.

Reply:
Yeah, but in the future when Rewind is available to all sites, we'll want to use this status to show upgrade messages as well as "Needs credentials" messages etc.